app/views/setup/welcome.html.slim

```diff
@@ -16,6 +16,55 @@ h1 Initial CaaSP Configuration
 
   .panel.panel-default
     .panel-heading.clearfix
+      h3.panel-title Cluster network settings
+      .btn-group.btn-group-sm.btn-group-toggle.pull-right data-toggle="buttons"
+        = label_tag :cluster_network_settings_toggle, nil, class: "btn btn-default", data: {toggle: "collapse", target: "#cluster-network-settings-panel"}
+          | Show
+
+    #cluster-network-settings-panel.panel-collapse.collapse
+      .panel-body
+        .form-group
+          = f.label :cluster_cidr, "Cluster CIDR"
+          .input-group
+            = f.text_field :cluster_cidr, value: @cluster_cidr, class: "form-control", required: true
+            span class="input-group-addon"
+              a data-toggle="tooltip" data-placement="left" title="Cluster CIDR"
+                i class='glyphicon glyphicon-info-sign'
+
+        .form-group
+          = f.label :cluster_cidr_min, "Cluster CIDR (lower bound)"
+          .input-group
+            = f.text_field :cluster_cidr_min, value: @cluster_cidr_min, class: "form-control", required: true
+            span class="input-group-addon"
+              a data-toggle="tooltip" data-placement="left" title="Cluster CIDR (lower bound)"
+                i class='glyphicon glyphicon-info-sign'
+
+        .form-group
+          = f.label :cluster_cidr, "Cluster CIDR (upper bound)"
+          .input-group
+            = f.text_field :cluster_cidr_max, value: @cluster_cidr_max, class: "form-control", required: true
+            span class="input-group-addon"
+              a data-toggle="tooltip" data-placement="left" title="Cluster CIDR (upper bound)"
+                i class='glyphicon glyphicon-info-sign'
+
+        .form-group
+          = f.label :cluster_cidr, "Cluster CIDR length"
+          .input-group
+            = f.text_field :cluster_cidr_len, value: @cluster_cidr_len, class: "form-control", required: true
+            span class="input-group-addon"
+              a data-toggle="tooltip" data-placement="left" title="Cluster CIDR length"
+                i class='glyphicon glyphicon-info-sign'
+
+        .form-group
+          = f.label :cluster_cidr, "Services CIDR"
```

Comment:
Oh - we'll need to add dns:cluster_ip here too, as that needs to be inside the services CIDR